frontend/public/components/nav/admin-nav.tsx

```diff
@@ -59,13 +58,12 @@ const apiExplorerStartsWith = ['api-explorer', 'api-resource'];
 
 const monitoringNavSectionStateToProps = (state) => ({
   canAccess: !!state[featureReducerName].get(FLAGS.CAN_GET_NS),
-  kibanaURL: state[monitoringReducerName].get(MonitoringRoutes.Kibana),
```

Comment:
Had to remove the kibanaURL in the Monitoring section, although Im not sure if thats ok? or is that being accounted when the `ConsoleLink` solution for the Logging URLs was discussed ?

Reply:
I think `ConsoleLink` is the right approach for the Kibana link. It wasn't an option when we first added the link in 4.1. Either way, we could look at logging in a follow on. At the moment, it's not actually broken, correct?

@jcantrill What do you think about creating a `ConsoleLink` CR to add Kibana to the application launcher menu in the console masthead? This would avoid RBAC problems adding the config map to `openshift-config-managed`

Reply:
+1 I think we've agreed on this approach.